ignite-generator/component/templates/component-test.js.template

```diff
@@ -0,0 +1,24 @@
+import test from 'ava'
+import React from 'react'
+import { shallow } from 'enzyme'
+import <%= name %> from '../../App/Components/<%= name %>'
+
+test('component exists', t => {
+  const wrapper = shallow(<<%= name %>  />)
```

Comment:
OK, but how would that work with multiple props passed in (or not) to the component?
eg. I pass in one prop to the component, it renders text of `miss piggy`, I pass in another prop and it renders a picture of kermit 🐸 ... Wouldn't you need two tests to cover that scenario and two wrapper declarations (for the different props).

I might be missing the point... 🎯